src/js/actions/index.js

```diff
@@ -1,28 +1,57 @@
 import fb from '../utils/fb';
 
+export const REQUEST_FETCH_ADSETS = 'REQUEST_FETCH_ADSETS';
+export const COMPLETE_FETCH_ADSETS = 'COMPLETE_FETCH_ADSETS';
+export const FETCH_ADSETS = 'FETCH_ADSETS';
+export const SELECT_ADSET = 'SELECT_ADSET';
 export const REQUEST_FETCH_ADACCOUNTS = 'REQUEST_FETCH_ADACCOUNTS';
 export const COMPLETE_FETCH_ADACCOUNTS = 'COMPLETE_FETCH_ADACCOUNTS';
 export const FETCH_ADACCOUNTS = 'FETCH_ADACCOUNTS';
 export const SELECT_ADACCOUNT = 'SELECT_ADACCOUNT';
 export const SET_FB_STATUS = 'SET_FB_STATUS';
 
-export const requestFetchAdaccounts = () => ({ type: REQUEST_FETCH_ADACCOUNTS });
-export const completeFetchAdaccounts = () => ({ type: COMPLETE_FETCH_ADACCOUNTS });
-export const fetchAdaccounts = () => (dispatch) => {
-  dispatch(requestFetchAdaccounts());
-  fb.api('v2.8/me/adaccounts', { fields: 'id,name' }, (response) => {
+export const requestFetchAdSets = () => ({ type: REQUEST_FETCH_ADSETS });
+export const completeFetchAdSets = () => ({ type: COMPLETE_FETCH_ADSETS });
+export const fetchAdSets = (accountId) => {
+  return (dispatch) => {
+    dispatch(requestFetchAdSets());
+    fb.api(`v2.8/${accountId}/adsets`, { fields: 'id,name', limit: 100 }, (response) => {
```

Comment:
I think parenthesis `()` are not necessary in this line and above line. It's okay if you want to do it.